libs/base/Data/SnocList.idr

```diff
@@ -30,6 +30,8 @@ export
 Cast (List a) (SnocList a) where
   cast xs = Lin <>< xs
 
+%transform "fastConcat" concat {t = SnocList} {a = String} = fastConcat . cast
```

Comment:
Can you pull this out into a separate PR (and get it merged sooner)?

Reply:
Now I see that was part of the system run PR and therefore will get merged sooner.